src/components/ui/form-github/GithubForm.tsx

```diff
@@ -19,33 +20,26 @@ export const GithubForm: React.FC<GithubFormProps> = ({
   return (
     <div className='card relative my-auto h-full bg-neutral text-neutral-content lg:w-1/2'>
       <div className='card-body items-center justify-center gap-5 text-center'>
-        <button
-          className='btn absolute right-0 top-0 m-2'
-          onClick={onClick}
-          {...(isLoading ? { disabled: true } : {})}
+        <div
+          className='tooltip tooltip-left absolute right-0 top-0'
+          data-tip='Código Markdown'
         >
-          Código Markdown
-          <svg
-            xmlns='http://www.w3.org/2000/svg'
-            fill='none'
-            viewBox='0 0 24 24'
-            strokeWidth={1.5}
-            stroke='currentColor'
-            className='size-5'
+          <button
+            className='btn btn-outline m-2 p-3'
```

Comment:
Evitar modificar los estilos por defecto de los botones de `daysiUI` ya que podría causar distorsion de dimensiones
```suggestion
            className='btn btn-outline m-2'
```